src/declarations/stencil-public-runtime.ts

```diff
@@ -561,12 +561,12 @@ export interface FunctionalComponent<T = {}> {
  * {@link FunctionalUtilities}).
  */
 export interface ChildNode {
-  vtag?: string | number | Function;
-  vkey?: string | number;
-  vtext?: string;
-  vchildren?: VNode[];
+  vtag?: string | number | Function | null;
+  vkey?: string | number | null;
+  vtext?: string | null;
+  vchildren?: VNode[] | null;
```

Comment:
Does widening this to `null` have any affect on Functional Components? Namely, does this affect the `FunctionUtilities#forEach` function? I think that was originally written under the assumption that this property would always be an array (even if it were empty), but I'm not 100% sure of the fallout of making this `null`.